shared/templates/rsyslog_logfiles_attributes_modify/tests/include_is_other_IncludeConfig_is_other_RainerLogClause.fail.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# platform = Red Hat Enterprise Linux 8,multi_platform_fedora,Oracle Linux 8,multi_platform_sle
+
+# Check rsyslog.conf with root user log from rules and
+# root user log from include() passes.
+
+source $SHARED/rsyslog_log_utils.sh
+
+{{% if ATTRIBUTE == "owner" %}}
+CHATTR="chown"
+{{% else %}}
+CHATTR="chgrp"
+{{% endif %}}
+
+USER_TEST=testssg
```

Comment:
We have to ensure this user/group is created.

Reply:
👍 should be ok in [7447c59](https://github.com/ComplianceAsCode/content/pull/9789/commits/7447c59d1b7a61b89e6c69d5570fd742149b7638)